src/voice_generator/__init__.py

```
@@ -0,0 +1,30 @@
+"""
+Core constants and environment checks for the Voice Generator package.
+
+Keeping this module tiny avoids pulling heavy dependencies into processes
+that only need the shared constants (e.g. docs builders, CI linters).
+"""
+
+import os
```

Ruff (0.8.2)

`os` imported but unused

(F401)